hiera.yaml

```diff
@@ -0,0 +1,16 @@
+---
+version: 4
+datadir: data
+hierarchy:
+  - name: 'Full version'
+    backend: yaml
+    path: "%{facts.os.name}-%{facts.os.release.full}"
```

Comment:
here we probably want `%{facts.os.name}-%{facts.os.release.major}` on my CentOS 7 this as is would be `CentOS-7.2.1511.yaml` which is a bit too specific to be usable and - at least for sane distros - major release boundary should be good enough?